android/app/src/main/java/com/wibblystuff/gear/MainActivity.java

```diff
@@ -24,6 +24,6 @@ protected void onCreate(Bundle savedInstanceState) {
      */
     @Override
     protected String getMainComponentName() {
-        return "main";
+        return "PocketGear";
```

Comment:
I'd like to keep it as `main` and change `PockerGear` to `main` in `AppDelegate.m`